docs/gen_ref_nav.py

```diff
@@ -24,7 +24,7 @@
         ident = ".".join(parts)
         print("::: " + ident, file=fd)
 
-    mkdocs_gen_files.set_edit_path(full_doc_path, path)
+    mkdocs_gen_files.set_edit_path(full_doc_path, Path("../") / path)
```

Comment:
I did not foresee the possibility that people would need to use `..` *and* even more so I'm surprised that it actually works very cleanly.
```pycon
>>> urllib.parse.urljoin('http://example.com/foo/bar/tree/master/docs/', '../src/stuff/hi.py')
'http://example.com/foo/bar/tree/master/src/stuff/hi.py'
```